cmd/manager/main.go

```diff
@@ -16,8 +16,11 @@ limitations under the License.
 package main
 
 import (
+	"fmt"
 	"os"
 
+	"github.com/kudobuilder/kudo/pkg/version"
+
```

Comment:
could we collapse this line?

Reply:
gofmt breaks out built-ins vs. external packages